test/test-utils.js

```diff
@@ -11,7 +11,7 @@ function bn (number) {
 
 describe('utils', function () {
   describe('addRange', () => {
-    it('adds ranges and merges if possible', async () => {
+    it('adds ranges and merges if possible', async (done) => {
```

Comment:
Hmm is this necessary?

Reply:
Hmm maybe not actually. There is an issue with the tests hanging that I was trying to fix, but this didn't seem to do the trick. I believe it is this test that is causing the hanging though. I can revert it for now

Reply:
I went ahead and reverted these changes